### ISSUE TITLE
`GM_unregisterMenuCommand` will cause all the script menus to disappear. `v4.18.0`

### ISSUE THREAD

**Issue by F9y4ng** [Tampermonkey/tampermonkey#1607]
(Please fill out the issue template with your details)

## Expected Behavior
fixed menu issues.

## Actual Behavior
`GM_unregisterMenuCommand` will cause all the script menus to disappear, even the menus of other scripts.

when use `GM_unregisterMenuCommand`:

![menu error](https://user-images.githubusercontent.com/15855068/193626294-67b11193-057a-4883-9f17-b892e838b338.png)

## Specifications

- Chromium: (i.e. 64.0.3282.140) chrome 106.0.5249.91 / Firefox 105.0.1
- TM: (i.e. 4.5.5648) 4.18.0
- OS: (i.e. MacOS 10.13.4) WIN 10

## Script

(Please give an example of the script if applicable.)

```js
// ==UserScript==
// @name         Test
// @namespace    http://tampermonkey.net/
// @version      0.1
// @description  try to take over the world!
// @author       You
// @match        *://*/*
// @icon         https://www.google.com/s2/favicons?sz=64&domain=google.com
// @grant              GM_registerMenuCommand
// @grant              GM_unregisterMenuCommand
// ==/UserScript==

(function() {
    'use strict';
    let test1 = GM_registerMenuCommand("test1", () => {
               alert("test1");
    });
    let test2 = GM_registerMenuCommand("test2", () => {
               alert("test2");
    });
   GM_unregisterMenuCommand(test1);
})();
```

**Comment by cooper1x** [comment]
me too

**Comment by A3148475824** [comment]
me too，Closing the script also causes it to disappear

**Comment by derjanb** [comment]
Should be fixed at 4.19.6170 (in review|[crx](https://tampermonkey.net/crx/tampermonkey_beta_current.crx)|[xpi](https://firefox.tampermonkey.net/firefox-current-beta.xpi))

Chrome/Edge users, please download the crx file linked above and drag and drop it to the extensions page `chrome://extensions`.
Firefox users please install the [BETA version](https://www.tampermonkey.net/index.php?browser=firefox) or check for BETA version updates at `about:addons` 

For a quick fix please export your settings and scripts as zip or (JSON) file at the "Utilities" tab and import it back at the fixed BETA version.

**Comment by F9y4ng** [comment]
> Should be fixed at 4.19.6170 (in review|[crx](https://tampermonkey.net/crx/tampermonkey_beta_current.crx)|[xpi](https://firefox.tampermonkey.net/firefox-current-beta.xpi))
> 
> Chrome/Edge users, please download the crx file linked above and drag and drop it to the extensions page `chrome://extensions`. Firefox users please install the [BETA version](https://www.tampermonkey.net/index.php?browser=firefox) or check for BETA version updates at `about:addons`
> 
> For a quick fix please export your settings and scripts as zip or (JSON) file at the "Utilities" tab and import it back at the fixed BETA version.

I tested Tampermonkey BETA 4.19.6170, and it was indeed fixed in Chrome and Firefox, thanks.
